molecule/focal-github/molecule.yml

```diff
@@ -0,0 +1,13 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: docker
+platforms:
+  - name: instance
+    image: ubuntu:20.04
```

Comment:
I wondering if splitting this into an `image` and `tag` would be overkill?

Reply:
I think that the schema for this file is controlled by the Molecule Docker driver (or whatever they refactored it into). Stupidly, that project [doesn't publish documentation anywhere](https://github.com/ansible-community/molecule-plugins/issues/20), so I can't confirm this. But basically, I don't think we can split this.

Regardless, what we _should_ do is use a shared base config, so we could avoid having to specify the image entirely (see how `ansible-consul` does it: https://github.com/ansible-community/ansible-consul/tree/master/molecule). We have an [issue open for that](https://github.com/AbletonAppDev/devtools/issues/4855), but it's in the purgatory/icebox column until we get time.

Reply:
Makes sense. I missed that this was part of `molecule`. Sorry for the noise!